### DISCUSSION TITLE
How is `%12v` expanded to the virtual env name?

### DISCUSSION THREAD

**Discussion by NightMachinery** [sindresorhus/pure#634] [General]
I am trying to figure out how the virtual env name is being added to my prompt so that I can change it manually. But I can't find the docs on how `%12v` is expanded to the virtual env name.

<img width="654" alt="image" src="https://user-images.githubusercontent.com/36224762/173039774-f44f9043-b74e-4628-9c44-6c889269aecb.png">

**Comment by mafredri** [comment]
I'm interested in how you ran into this issue. We used `psvar[12]` to (try to) avoid collisions 😄.

From zsh manual:

```
       psvar <S> <Z> (PSVAR <S>)
              An array (colon-separated list) whose elements can be used in PROMPT strings.  Setting psvar also sets PSVAR, and vice versa.
```


See here for how we set it: https://github.com/sindresorhus/pure/blob/3b2606db2e7eb6f05c67b229b5e3c68a1c3947a6/pure.zsh#L216-L237

**Reply by NightMachinery** [reply]
Thanks. There is no conflict on my part, I just wanted to change this venv, as I sometimes manually update my PATH and the venv becomes invalid.
